src/app/GlobalStyles.tsx

```diff
@@ -5,6 +5,10 @@ const GlobalStyle = createGlobalStyle`
   box-sizing: border-box;
 }
 
+:root {
+  --background-title: #0000ffff;
+}
+
 body {
   margin: 10px;
     font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
```

Comment:
Formatting looks strange, is stylelint working?

Reply:
Could also be prettier that's not working :)